.github/workflows/phpunit.yml

```diff
@@ -36,7 +36,7 @@ env:
 jobs:
   main:
     name: PHP ${{ matrix.php-versions }} - ${{ matrix.db-platforms }}
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
this can use 24.04 as it is now the latest image

Reply:
I was hoping to keep it aligned with our current version in the main repo:
https://github.com/codeigniter4/CodeIgniter4/blob/develop/.github/workflows/reusable-phpunit-test.yml#L64